circle.yml

```diff
@@ -23,8 +23,11 @@ dependencies:
         "$CIRCLE_PROJECT_USERNAME" 
         "$CIRCLE_PROJECT_REPONAME" 
         "$CIRCLE_BUILD_URL"
-        > recommendation/static/version.json
-    - cp recommendation/static/version.json $CIRCLE_ARTIFACTS
+        > version.json
+    - cp version.json $CIRCLE_ARTIFACTS
+
+    # for serving /__version__ from the web
+    - cp version.json recommendation/static/version.json
```

Comment:
I'm not sure. I think we'd need to run a `ln -s` in the Dockerfile but figured this way it's a little clearer. I I can change it to a symlink if you prefer.